### ISSUE TITLE
Alt attribute for images

### ISSUE THREAD

**Issue by jim-rosales** [phdigimedia/teacozy#2]
It's always a good idea to add the alt attribute to every image you use on your site. The alt attribute is used by screen readers for user that are visually impaired to describe the content on your page. The alt attribute is also used as a backup incase an error occurs and your image cannot be displayed. 

Here is a [gist](https://gist.github.com/jim-rosales/2d3d6e04d880b8365cc7933009106446) which shows the use of the alt attribute in several images. 

https://github.com/phdigimedia/teacozy/blob/81c069832dbc0f05259e87f905dcdd7974e4e172/index.html#L33-L49
